pubspec.lock

```diff
@@ -42,7 +42,7 @@ packages:
       name: bloc
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.0.0"
```

Comment:
can you also do these bumps in a separate PR, since this is not related to the README.

Reply:
can I say "next time"? ;)
It's a really small bump

Reply:
Can I respond with 🤦 ? i'm thinking...